keras_nlp/layers/transformer_decoder.py

```diff
@@ -218,6 +217,7 @@ def call(
         self_attention_cache_update_index=None,
         cross_attention_cache=None,
         cross_attention_cache_update_index=None,
+        use_causal_mask=True,
```

Comment:
default to True in order to not break existing behavior

Reply:
sgtm. I think default to `True` is probably makes sense here regardless. The most beaten path for these decoder layers will be with the causal mask applied.